int32 hz
int32  N
float32[] l_arr
float32[] L_arr
---
bool checksum